src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE = 7;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure(int value) {
+        int rndInt = random.nextInt(MAX_VALUE);
+
+        switch (value) {
+            case 1:
+                return new Circle(colorSupplier.getRandomColor(), random.nextInt(rndInt));
```

Comment:
```suggestion
                return gerRandomCircle();
```
lets divide logic into smaller methods, 
lets create a separate **private** method to get each figure